lua/ulx/modules/sh/cfc_speed.lua

```diff
@@ -0,0 +1,23 @@
+CFCUlxCommands.speed = CFCUlxCommands.speed or {}
+local cmd = CFCUlxCommands.speed
+
+CATEGORY_NAME = "Fun"
+                    
+function cmd.speedPlayers( callingPlayer, targetPlayers, amount )
+    for _, ply in pairs( targetPlayers ) do
+        local inPvp = hook.Run( "CFC_ULXCommands_CanApplySpeedModifier", ply )
+        --false = in pvp
+        if inPvp ~= false then
+            ply:SetWalkSpeed( 2 * amount )
+            ply:SetRunSpeed( 4 * amount )
```

Comment:
Are these two values important? Should they be configurable be the user, or maybe at the top of the file / in a convar?

Reply:
They're the default movement speed values at 100% (100x2 and 100x4)